src/components/SubHeader/index.tsx

```diff
@@ -7,10 +7,17 @@ import { size, subheaderHeight } from "constants/tokens";
 const { gray } = palette;
 
 /** TODO: EVG-17534 */
-const SubHeader = () => (
+interface SubHeaderProps {
+  isUploadedLog: boolean;
+}
+const SubHeader: React.FC<SubHeaderProps> = ({ isUploadedLog }) => (
   <Container>
     <StyledIcon glyph="EvergreenLogo" />
-    <StyledLink href="/test">Task Page</StyledLink>
+    {isUploadedLog ? (
+      <span>Thanks for uploading</span>
```

Comment:
This seems somewhat irrelevant to pin to the page as a non-dismissible banner. Maybe showing the filename of the uploaded log would be helpful here?

Reply:
Good idea i put it on the agenda for next design sync but filename sgtm